activefires_pp/tests/conftest.py

```diff
@@ -63,20 +63,10 @@
 
 output_dir: /path/where/the/filtered/results/will/be/stored
 
-filepath_detection_id_cache: /path/to/the/detection_id/cache/fire_detection_id_cache.txt
+#filepath_detection_id_cache: /path/to/the/detection_id/cache/fire_detection_id_cache.txt
```

Comment:
do we need to keep this comment?

Reply:
Yes, you are right. It is already available in the example yaml file, and here I removed it in order to make this file path on the fly in some of the tests...